css/style.css

```diff
@@ -68,6 +281,6 @@ body {
   text-transform: uppercase;
 }
 
-.footer-navbar {
-  /* color: #ffffff; */
-}
\ No newline at end of file
+/* .footer-navbar {
+  color: #ffffff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654147)